test/sqllogictest/cockroach/union.slt

```diff
@@ -257,7 +257,7 @@ CREATE TABLE b (a INTEGER PRIMARY KEY)
 
 query I
 SELECT * FROM a UNION ALL SELECT * FROM b
-
+----
```

Comment:
Are there not dozens more examples of this in the cockroach files? I seem to recall it being a fairly common pattern, and so one we might need to support.

Reply:
I just checked and there are no others. Stuff with `query error` often doesn't have `----` but that's a different path in the parser.

Reply:
👍